### PR TITLE
Improve list indicator styles

### PR DESCRIPTION
**Pull request by txsmith**
screenshot from 2016-05-03 15 50 47

### CODE REVIEW

**Comment by eanker** [comment]
Why did you change the project indicators, I liked the old style more (see image).
image

File: src/list-items/column-list-item.html [hunk 1]
```diff
@@ -30,18 +30,29 @@
...
         flex: 1;
         flex-direction: column;
-        justify-content: space-between;
+      }
+      ::content > .info {
+        max-width: 100px;
+      }
+
+      ::content iron-icon {
+        /*width: 20px;
```
Comment (TimvdLippe) [inline src/list-items/column-list-item.html:53]: Left-over styling.

File: src/projects/projects-page.html [hunk 2]
```diff
@@ -33,10 +33,12 @@
         flex: 1;
       }
       project-overview-page {
-        max-width: 400px;
         overflow-y: scroll;
         overflow-x: hidden;
       }
+      project-pull-requests {
+        max-width: 630px;
```
Comment (TimvdLippe) [inline src/projects/projects-page.html:40]: This does not look good on my screen. The previous on `project-overview-page` was better imo.

Comment (txsmith) [inline src/projects/projects-page.html:40]: I have extracted an issue for this (#16)

**Comment by TimvdLippe** [comment]
LGTM

**Pull request merged**